0-ad-history-mod-package/simulation/templates/units/spart/siege_oxybeles_packed.xml

```diff
@@ -4,6 +4,11 @@
     <Civ>spart</Civ>
     <SpecificName>Oxybelḗs</SpecificName>
     <Icon>units/hele/siege_oxybeles_packed.png</Icon>
+    <History>
+        Engineers in Ancient Greece tried to improve the traditional bow and developed the gastraphetes, a crossbow. However, they wanted an even more powerful weapon, and in the 4th century B.C., the oxybeles was invented.
+        The oxybeles was just a very big gastraphetes (crossbow), firing big arrows or javelins, mounted onto a tripod with a swivel joint. This enabled it to be easily rotated in all directions. Just like the gastraphetes, the oxybeles was made from horn, wood, and sinew. Even though it had greater power, range, and accuracy than most weapons at that time, it was not strong enough to take down buildings or walls. But still, the oxybeles was a very dangerous weapon, it could pierce any soldier's shield and armor from hundreds of meters away.
+        The Spartans rarely used siege weapons and preferred hand-to-hand combat. It can still be assumed they used the oxybeles to some extent since it originated from and was frequently used in ancient Greece.
```

Comment:
```suggestion
        The Spartans rarely used siege weapons, perferring hand-to-hand combat. It can still be assumed they used the oxybeles to some extent since it originated from and was frequently used in ancient Greece.
```